.gitignore

```diff
@@ -1,8 +1,13 @@
-data/*
-example_data/*
-example/test_data/*
-garnet_data
-example/TF_regress_example.ipynb
+# Large motif data files
+garnet_data/motifMap_hg19*
+garnet_data/cisBP_hg19
+
+# Files generated by example notebook
+garnet_data/reference/ucsc_reference.hg19.tss.bed
+garnet_data/cisBP_hg19_SLIM/garnetDB_cisBP.hg19.LOD.10kb.chr1_SLIM.tsv
```

Comment:
instead of gitignoring specific files, can we gitignore directories where we put all the files we want to gitignore? I think I was using `data/*` for this..

Reply:
Yea though about that, but it was sort of making my workflow messy. I would like to keep all files related to garnet file generation in one folder, and individual datasets in another, since that's what I would use personally. But we can change it so all files needed for the example are in one directory.